### PR TITLE
Fix linting errors

### PR DESCRIPTION
**Pull request by zivkovicmilos**
# Description

This PR resolves the linting errors outlined in the latest version of  `golangci-lint`, as well as switches the import parser to `goimports` instead of `gofmt`

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [x] I have tested this code with the official test suite
- [ ] I have tested this code manually

### CODE REVIEW

File: blockchain/subscription.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func (m *MockSubscription) GetEvent() *Event {
 func (m *MockSubscription) Close() {
 }
 
-/////////////////////////
+// ///////////////////////
```
Comment (Kourin1996) [inline blockchain/subscription.go]: What is that?

**Review by Kourin1996 — APPROVED** [review]
LGTM

File: blockchain/subscription.go (hunk 1 above)
Comment (zivkovicmilos) [reply blockchain/subscription.go]: I honestly have no clue 😄 

Removed:

[1fa3f31](https://github.com/0xPolygon/polygon-edge/pull/675/commits/1fa3f318a648e5d8462831dac77970cc966a56a9)

File: consensus/ibft/verifier.go [hunk 2]
```diff
@@ -4,11 +4,12 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+
 	protoIBFT "github.com/0xPolygon/go-ibft/messages/proto"
 	"github.com/0xPolygon/polygon-edge/types"
 )
 
-//	Verifier impl for go-ibft
+// Verifier impl for go-ibft
```
Comment (Aleksao998) [inline consensus/ibft/verifier.go:13]: why empty line

**Review by dbrajovic — APPROVED** [review]

**Review by 0xAleksaOpacic — APPROVED** [review]

**Pull request merged**